compact/tracker.xml

```diff
@@ -133,16 +133,17 @@
   </documentation>
   <plugins>
     <plugin name="epic_FileLoader">
-      <arg value="cache:$DETECTOR_PATH:/opt/detector"/>
+      <arg value="cache:$DETECTOR_PATH"/>
```

Comment:
Was there a reason for removing this? It is required to find the material maps in the container.

Reply:
Not sure why that is changed. I did not explicitly change it, oh maybe it was from sourcing my local build directory? Anyhow I think keeping it as is ($DETECTOR_PATH:/opt/detector ) is fine

Reply:
Fixed